.github/policies/resourceManagement.yml

```diff
@@ -775,6 +785,7 @@ configuration:
         then:
         - mentionUsers:
             mentionees:
+            - jaredpar
```

Comment:
All instances of jaredpar should be replaced with phil-allen-msft (here and in the area-owners)